teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, name, specialization, parent_permission: true)
+    super(age: age, name: name, parent_permission: parent_permission)
+    @age = age
+    @name = name
```

Comment:
- [ OPTIONAL ] I think you don't need to add defualt values here to the name, age and parent permission so kindly just pass the attribute itself like this

and remove @age = age, @name = name because they are already intialized in the base class 😊🚀💯💯

>  super(age, name, parent_permission)